exo/api/chatgpt_api.py

```diff
@@ -119,31 +138,37 @@ def generate_completion(
     choice = completion["choices"][0]
     if object_type.startswith("chat.completion"):
         key_name = "delta" if stream else "message"
-        choice[key_name] = {"role": "assistant", "content": tokenizer.decode(tokens)}
+        choice[key_name] = {"role": "assistant",
```

Comment:
It looks like your linter is being aggressive with line lengths. We probably need a linter in general but lets leave that to a different PR.